omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml

```diff
@@ -1741,9 +1823,57 @@
                                                         </repeat>
                                                     </div>
                                                 </div>
-                                            </div>
                                         </div>
                                         <div class="card">
+                                                <div class="card-header">
+                                                    <label>Other Medications</label>
+                                                </div>
+                                                <div class="card-body">
+                                                    <div>
+                                                        <repeat>
+                                                            <template>
+                                                                <obsgroup groupingConceptId="160741">
```

Comment:
@solemabrothers You need additional grouping concepts otherwise the content in here will be overwritten